junit-jupiter-params/src/main/java/org/junit/jupiter/params/provider/CsvFileSource.java

```diff
@@ -69,4 +69,10 @@
 	 */
 	char delimiter() default ',';
 
+	/**
+	 * The number of lines to skip, if the CSV files have header rows.
```

Comment:
This makes it sound as if we only skip header rows when in fact we skip any row.

Reply:
Okay, what about this then?
````
/**
* The number of lines to skip parsing. Commonly used for skipping header rows in the CSV files.
*/
````